services/nvidia-gpu-operator/metadata.yaml

```diff
@@ -0,0 +1,33 @@
+displayName: NVIDIA GPU Operator
+description: NVIDIA Data Center GPU Manager (DCGM) manages and monitors NVIDIA datacentre GPUs in cluster environments.
```

Comment:
TODO this metadata needs to be rewritten -> https://jira.d2iq.com/browse/D2IQ-92963